### ISSUE TITLE
Ruleset Generator Does not properly import base ruleset

### ISSUE THREAD

**Issue by pooki3bear** [nccgroup/Scout2#169]
The Ruleset Generator does not properly import the default ruleset and policies generated from it lack most of the Scout2 defaults.

![image](https://user-images.githubusercontent.com/24210410/27609456-ac018b70-5b3f-11e7-9c9b-6f744b112636.png)

![image](https://user-images.githubusercontent.com/24210410/27609479-c1d87ca6-5b3f-11e7-87dc-4c376fa29e96.png)

No matter which ruleset I try to import as the base, there is no difference in the content of HTML wizard page regardless if there was an error from the python script or not. 

As an example there are 23 S3 rules in the default ruleset https://github.com/nccgroup/Scout2/blob/master/AWSScout2/rules/data/rulesets/default.json

When I open ruleset-generator.html there are only 6 rules available to edit.

**Comment by l01cd3v** [comment]
Thanks, I'll look into this.

**Comment by l01cd3v** [comment]
I _think_ that the code on the "new-ruleset-generator" branch works and addresses this issue. Would you mind giving it a try?

I would like to further test it before I merge into master as I made a fairly large amount of changes, so the merge may not happen for a little while as I have limited time to work on the tool right now.

https://github.com/nccgroup/Scout2/tree/new-ruleset-generator

**Comment by pooki3bear** [comment]
Hi, I did a fresh pull and check'd out the new-ruleset-generator branch. 

It lists opinel>=3.1.0,<4.0.0 in requirements and i dunno how install. 

I tried just commenting out and running with my current version of 3.0.15

I tried installing https://github.com/nccgroup/opinel master from source instead of pip but problems with setup.py

Should I be using a dev branch of Opinel to test this branch of ruleset-generator?
